src/Compilers/CSharp/Portable/Binder/UsingStatementBinder.cs

```diff
@@ -68,13 +68,12 @@ internal override BoundStatement BindUsingStatementParts(DiagnosticBag diagnosti
 
             Debug.Assert((object)disposableInterface != null);
             bool hasErrors = ReportUseSiteDiagnostics(disposableInterface, diagnostics, hasAwait ? _syntax.AwaitKeyword : _syntax.UsingKeyword);
-
-            Conversion iDisposableConversion = Conversion.NoConversion;
+            _ = Conversion.NoConversion;
             BoundMultipleLocalDeclarations declarationsOpt = null;
             BoundExpression expressionOpt = null;
             AwaitableInfo awaitOpt = null;
             TypeSymbol declarationTypeOpt = null;
-
+            Conversion iDisposableConversion;
```

Comment:
really don't like that not only did we split, but we separated.

Reply:
This is moving the declaration near the next closest reference...

Reply:
understood.  just giving you my gut reaction :)